components/communication-template/action-template.tsx

```diff
@@ -0,0 +1,195 @@
+import { useState } from 'react'
```

Comment:
Component that handles ONLY the instance-wide action template config/association, independent of the revokeCredentials action.